benchmarks/quad_hexagon.py

```diff
@@ -14,9 +14,9 @@ def time_open_grid(self):
         """Time to open a `Grid`"""
         ux.open_grid(grid_path)
```

Comment:
how does this measure time etc.?

Reply:
https://asv.readthedocs.io/en/stable/writing_benchmarks.html#timing